---
# Documentation: https://wowchemy.com/docs/managing-content/

title: 'Considering external information to improve the phylogenetic comparison of
  microbial communities: a new approach based on constrained Double Principal Coordinates
  Analysis (cDPCoA)'
subtitle: ''
summary: ''
authors:
- St√©phane Dray
- Sandrine Pavoine
- Daniel Aguirre de Carcer
tags: []
categories: []
date: '2015-01-01'
lastmod: 2022-09-27T14:50:48+02:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-09-27T13:40:08.492826Z'
publication_types:
- '2'
abstract: The use of next-generation sequencing technologies is revolutionizing microbial
  ecology by allowing a deep phylogenetic coverage of tens to thousands of samples
  simultaneously. Double Principal Coordinates Analysis (DPCoA) is a multivariate
  method, developed in community ecology, able to integrate a distance matrix describing
  differences among species (e.g. phylogenetic distances) in the analysis of a species
  abundance matrix. This ordination technique has been used recently to describe microbial
  communities taking into account phylogenetic relatedness. In this work, we extend
  DPCoA to integrate the information of external variables measured on communities.
  The constrained Double Principal Coordinates Analysis (cDPCoA) is able to enforce
  a priori classifications to retrieve subtle differ- ences and (or) remove the effect
  of confounding factors. We describe the main principles of this new approach and
  demonstrate its usefulness by providing application examples based on published
  16S rRNA gene data sets.
publication: '*Molecular Ecology Resources*'
doi: 10.1111/1755-0998.12300
---
